layouts/page/resources.html

```diff
@@ -25,41 +25,98 @@ <h2 class="text-center text-primary fw-lighter">
-                                {{ $randomHue := mod (int (printf "0x%s" (substr (crypto.MD5 .) 0 5))) 360 }}
-                                <div class="col">
-                                    <a class="text-decoration-none" href="{{ printf $fileLink . }}" target="_blank">
-                                        <div class="card border-1 rounded-0">
-                                            <div class="card-body d-flex flex-column justify-content-center align-items-center"
-                                                 style="--hue: {{ $randomHue }};">
-                                                <div class="text-center thumbnail"
-                                                     style="height: 70%; --thumbnail: url('{{ absURL (printf "resources/%s.png" .) }}'); --default-thumbnail: url('{{ absURL "default-thumbnail.png" }}');">
-                                                </div>
-                                                <div class="mt-4" style="height: 30%;">
-                                                    <h6 class="card-subtitle text-body-secondary clamp"
-                                                        style="--clamp: 3;" title="{{ $filename }}">
-                                                        {{ $filename }}
-                                                    </h6>
+                    <div class="d-flex flex-column flex-sm-row align-items-center justify-content-between pt-4 pb-1 mb-2">
+                        <h6 class="text-uppercase mb-0">
+                            -&nbsp;{{- .group_name -}}&nbsp;-
+                        </h6>
+                        <a href="{{ .folder_link }}" target="_blank" class="text-muted ms-auto">View more...</a>
+                    </div>
+                        {{ $group_name := .group_name }}
+                        
+                        <!-- Handling subjects where there are smaller NAMED categories inside (subsections) -->
+                        <!-- Refer to yaml file for clarity in categorized and non-categorized subject sections  -->
+                        {{ if .has_subsection }}
+                            <div class="d-grid pb-2">
+                                {{ range .subsection }}
+                                    <!-- Each category in a subject is a dropdown menu (accordion component) by it self -->
+                                    <!-- Each dropdown ust have unique id, even in different tabs -->
+                                    <div class="accordion">
+                                        <div class="accordion-item" id={{ $accordianCounter }}>
+                                            <h2 class="accordion-header" id={{ print "heading-" $accordianCounter }}>
+                                                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target={{ print "#body-" $accordianCounter }} aria-expanded="true" aria-controls={{ print "body-" $accordianCounter }}>
```

Comment:
```suggestion
                                                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target={{ print "#body-" $accordionCounter }} aria-expanded="true" aria-controls={{ print "body-" $accordionCounter }}>
```

Expected: Collapsed btn has gray text, open then green
Acutal: A btn that has not clicked before is green, open still green, closed then turn back gray

If not mistaken adding `collapsed` will do lah (by inspecting the html), u check "join us" page accordion

One more thing is the border, kinda wierd frankly speaking (which is the default behavior of bs), u check "join us" page as well. I forgot how did i change...